### PR TITLE
Allow custom Google Play tracks to be defined for package names.

### PR DESCRIPTION
**Pull request by pocmo**

### CODE REVIEW

File: mozapkpublisher/push_apk.py [hunk 1]
```diff
@@ -76,6 +76,11 @@ def run(self):
             for apk_path in apks_paths
         }
 
+        for package_name in map(lambda x: x['package_name'], apks_metadata_per_paths.values()):
```
Comment (JohanLorenzo) [inline mozapkpublisher/push_apk.py]: Nit: the concept is great. A more pythonic way to write it: 

```py
for package_name in [metadata['package_name'] for metadata in apks_metadata_per_paths.values()]:
```

File: mozapkpublisher/common/googleplay.py [hunk 2]
```diff
@@ -25,12 +25,25 @@
 
 # Google play has currently 4 tracks. Rollout deploys
 # to a limited percentage of users
-TRACK_VALUES = ('production', 'beta', 'alpha', 'rollout', 'internal')
+TRACK_VALUES = ['production', 'beta', 'alpha', 'rollout', 'internal']
```
Comment (JohanLorenzo) [inline mozapkpublisher/common/googleplay.py]: Nit: Let's rename this variable to `_DEFAULT_TRACKS`. We don't need to expose that variable anymore. The only usage was there: https://github.com/mozilla-releng/mozapkpublisher/pull/66/files#diff-45a657627129bc0d2b7ae44372d28670L32

File: mozapkpublisher/common/googleplay.py [hunk 3]
```diff
@@ -25,12 +25,25 @@
...
+ADDITIONAL_TRACK_VALUES = {
+    'org.mozilla.focus': ['nightly'],
+    'org.mozilla.klar': ['nightly']
+}
 
 logger = logging.getLogger(__name__)
 
 
+def get_valid_track_values_for_package(package_name):
+    return TRACK_VALUES + ADDITIONAL_TRACK_VALUES.get(package_name, [])
```
Comment (JohanLorenzo) [inline mozapkpublisher/common/googleplay.py]: :+1:

File: mozapkpublisher/test/test_push_apk.py [hunk 4]
```diff
@@ -292,6 +294,30 @@ def test_do_not_contact_google_play_flag_does_not_request_google_play(monkeypatc
     # authentication step
 
 
+def test_custom_google_play_track(edit_service_mock, monkeypatch):
```
Comment (JohanLorenzo) [inline mozapkpublisher/test/test_push_apk.py]: :+1: nice test!

**Comment by JohanLorenzo** [comment]
By the way, when do you want this PR deployed in production?

**Comment by pocmo** [comment]
Updated the PR!

> By the way, when do you want this PR deployed in production?

I need to setup the custom track with Sylvestre. After that I want to update our Nightly script to build and upload GeckoView builds to the new track.

**Comment by JohanLorenzo** [comment]
Alright! Please ping me whenever you want it deployed, then.

**Comment by JohanLorenzo** [comment]
This successfully passed in https://tools.taskcluster.net/groups/WnGEuMTCTC2fc3R0t0oF0Q/tasks/bPKQz5k7QMWf8ejJWE3NGQ/runs/0/artifacts \o/

**Pull request merged**